pouta_blueprints/static/js/services/AuthService.js

```diff
@@ -57,6 +59,25 @@ app.factory('AuthService', ['$q', 'localStorageService', 'Session', 'Restangular
             return false;
         },
 
+        isGroupOwnerOrAdmin : function() {
```

Comment:
trusting the localstorage makes it possible for the user to ruin their user experience by trying to hack things but i suppose this is acceptable as long as the checks in the views are valid.